### PR TITLE
zephyr's admin mark PR

### PR DESCRIPTION
**Pull request by jjpark-kb**
## About The Pull Request
https://github.com/Skyrat-SS13/Skyrat-tg/pull/7462
just has the PR back up again

Adds the ability for admins to assign special little icons to players to denote them.
image

Host Approved.

## Changelog
:cl: ZephyrTFA
admin: Player Marks, accepts any emoji, find them in the Player Panel and the Who verb
/:cl:

### CODE REVIEW

**Comment by FlamingLily** [comment]
Me when I admin note

😳 FlamingLily

File: modular_skyrat/modules/admin/code/player_alerts/preferences.dm [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/datum/preferences
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/admin/code/player_alerts/preferences.dm]: Woah tiger, we have a preferences file.

Comment (jjpark-kb) [reply modular_skyrat/modules/admin/code/player_alerts/preferences.dm]: you aren't my preference, stop pref breaking me right now.

Comment (GoldenAlpharex) [reply modular_skyrat/modules/admin/code/player_alerts/preferences.dm]: You still gotta fix this @jjpark-kb

**Comment by hugmeorelse** [comment]
Update? @jjpark-kb <3

**Comment by jjpark-kb** [comment]
The intent is to have modular modules. Making changes to another module makes it technically nonmodular in a sense.

**Comment by ZephyrTFA** [comment]
*insert surprised pikachu face meme*

**Comment by jjpark-kb** [comment]
Its gotta stay like this.

**Comment by Iamgoofball** [comment]
Why do we need this?

**Comment by ORCACommander** [comment]
isn't there a preexisting verb a player can click that identifies all admin ckeys that are active in the round?

**Comment by GoldenAlpharex** [comment]
So, I tried this out in-game, and it both doesn't seem to do anything, and there needs to be some more work put into the functionality, there needs to be some checks done to the input that just early returns if it's null, for instance.

**Comment by ZephyrTFA** [comment]
> So, I tried this out in-game, and it both doesn't seem to do anything, and there needs to be some more work put into the functionality, there needs to be some checks done to the input that just early returns if it's null, for instance.

I have confirmed this to work so I'm not quite sure what you mean here.

**Comment by GoldenAlpharex** [comment]
> > So, I tried this out in-game, and it both doesn't seem to do anything, and there needs to be some more work put into the functionality, there needs to be some checks done to the input that just early returns if it's null, for instance.
> 
> I have confirmed this to work so I'm not quite sure what you mean here.

I gave myself an emoji and was unable to view it anywhere.
Hitting cancel when entering a new emoji overrides the previous selection with None.
There didn't seem to be any log nor feedback that it had been applied.
It's kind of in an inconvenient place, I know the vast majority of our staff doesn't use the Who verb simply because pressing F6 is faster and more effective for what we need.
There's no easy-to-access list of what those emojis can be for admins.

**Comment by tf-4** [comment]
This is currently non-functional and seems to be inactive, feel free to re-open when ready.

**Pull request closed without merging**